Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -184,6 +209,52 @@ std::pair<size_t, size_t> VertexManagerBase::ResetFlushAspectRatioCount()
   return val;
 }
 
+static void SetSamplerState(u32 index, bool custom_tex)
```

Comment:
I wonder what was the reason for splitting this from the state Generate function? But I'm fine with both ways.

Reply:
I wanted to keep the Generate function dependent on bpmem only, and not have any config settings influencing it. That way, the UIDs are not config/system-dependent. There's probably a better place for this logic, but not in the backends like before IMO.

Reply:
Yeah, ofc, it must not be within the backends ;)